library/crates/Cargo.toml

```diff
@@ -9,6 +9,16 @@ path = "" # ignored by cargo raze
 antlr-rust = "0.2.0"
```

Comment:
These dependencies and aliased targets are used by TypeDB Client Rust and the Protobuf compiler that TypeDB Protocol will use